README.md

```diff
@@ -640,7 +640,7 @@ Table of Contents
   * [pubnub.com](https://www.pubnub.com/) - Swift, Kotlin and React messaging at 1 million transactions each month. Transactions may contain multiple messages.
   * [eyeson API](https://developers.eyeson.team/) - A video communication API service based on WebRTC (SFU, MCU) to build video platforms. Allows RealTime Data Injection, Video Layouts, Recordings, a fully featured hosted web UI (quickstart) or packages for custom UIs. Has a [free tier for developers](https://apiservice.eyeson.com/api-pricing) with 1000 meeting minutes a month.
   * [Upstash Kafka](https://upstash.com/kafka) - Serverless Kafka Cloud offering with per-request-pricing. It has a free tier with max 10000 messages per day.
-
+  * [Scramjet Cloud Platform Beta](https://www.scramjet.org/#join-beta) - An end-to-end stream processing platform in free beta and offering 15 petabyte-seconds of free compute after the beta ends.
```

Comment:
I think to add such a beta program we need to know details of its eventual pricing, things tend to get orphaned on this list, so adding a beta here where the benefits are known only to apply to beta users would just leed to stale data soon.

Reply:
Should I add this to the wording?

Reply:
https://docs.scramjet.org/platform/pricing - this is the pricing calculator on our page. It reflects the above pricing exactly.

If you have any remarks though, I'd be happy to implement. We're very early stage, so all feedback is really needed. :)

Reply:
OK, seems ok but looks like there is a conflict in the branch, do you mind rebasing?

Reply:
No probs. :)

Reply:
Ready. :)